### ISSUE TITLE
Add section about company values

### ISSUE THREAD

**Issue by PaulRBerg** [sablier-labs/v2-docs#98]
See [this](https://docs.0xsplits.xyz/#values).

**Comment by smol-ninja** [comment]
Do you have content for this? I thought of adding the Core Values from the [hiring notion](https://sablier.notion.site/Careers-at-Sablier-d3771d8eefbf44a8a8428436fb950a1d) but the ones that you shared in the description are particularly related to engineering and product building. So the hiring ones do not fit here. We also follow the same Values mentioned in [the link](https://docs.splits.org/#values) so I can take inspiration from them and write our own. 

Let me know what you think.

**Comment by PaulRBerg** [comment]
hmm what if we just shared our [coding practices](https://www.notion.so/sablier/Sablier-Coding-Practices-Contracts-2f5920cf497646b4a2ea982c1bacb5a8) instead?

Basically, move that document from Notion to this Docusaurus site.

**Comment by PaulRBerg** [comment]
Yeah, that's a fair point. In this case, let's add both the values and the coding practices as two new documents under "Concepts", right after "Security".

<img width="297" alt="SCR-20241211-nban" src="https://github.com/user-attachments/assets/2b158c98-3755-4d26-a8ec-a111f447761c">
